packages/core/stories/form-field/form-field.doc.stories.mdx

```diff
@@ -28,12 +28,18 @@ All examples demonstrate Salt form controls inside the Form Field, including Inp
 
 ## Form Controls
 
-Form field can wrap any Salt UI control that is typically found in a form, such as Input, Checkbox and Radio Button components.
+Form field can wrap any Salt UI control that is typically found in a form, such as Input, Multiline Input, Checkbox and Radio Button components.
 
 <Canvas>
   <Story id="core-form-field--with-controls" />
 </Canvas>
 
+When using Form Field with Multiline Input, the `question` prop can be used on the form label and the `fullBorder` prop can be set on Multiline Input to style the field as a question.
```

Comment:
I assume this isn't worded best - needs to be reworded properly in accordance with your design guidance @bhoppers2008